src/portal/src/app/project/webhook/webhook.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="row">
+  <div *ngIf="!showCreate">
+    <div>
+        <div class="row flex-items-xs-between rightPos">
+            <div class="flex-xs-middle option-left">
+              <div>
+                <span style="font-weight: bold;">Webhook endpoint</span>: {{endpoint}}
+                <button class="btn btn-link" (click)="openAddWebhookModal()">{{'WEBHOOK.EDIT_BUTTON' | translate}}</button>
+              </div>
+              <div [ngSwitch]="isEnabled">
+                <button *ngSwitchCase="false" class="btn btn-link" (click)="switchWebhookStatus(true)">{{'WEBHOOK.ENABLED_BUTTON' | translate}}</button>
+                <button *ngSwitchCase="true" style="color: #cf4c23;" class="btn btn-link" (click)="switchWebhookStatus(false)">{{'WEBHOOK.DISABLED_BUTTON' | translate}}</button>
```

Comment:
please use class to define this style: color: #cf4c23;

Reply:
fixed